pyneuroml/swc/LoadSWC.py

```diff
@@ -334,3 +334,22 @@ def load_swc(filename: str) -> SWCGraph:
             tree.add_node(node)
 
     return tree
+
+
+def export_swc(graph: SWCGraph, filename: str) -> None:
```

Comment:
I'd add this to the `SWCGraph` class as `export_to_swc_file`. It's still SWC graph related functionality.